stac_ipyleaflet/stac_discovery/stac.py

```diff
@@ -173,12 +199,14 @@ def stac_bounds(url=None, collection=None, item=None, titiler_stac_endpoint=None
         if isinstance(titiler_stac_endpoint, str):
             r = make_get_request(f"{titiler_stac_endpoint}/stac/bounds", kwargs).json()
         else:
-            r = make_get_request(titiler_stac_endpoint.url_for_stac_bounds(), kwargs).json()
+            r = make_get_request(
+                titiler_stac_endpoint.url_for_stac_bounds(), kwargs
+            ).json()
 
         bounds = r["bounds"]
         return bounds
 
-    # QUESTION: Is this being or planning to be used? 
+    # QUESTION: Is this being or planning to be used?
```

Comment:
📝 : I guess this is being referenced within another file, I added this but I guess it can be removed